src/App.tsx

```diff
@@ -41,6 +40,10 @@ function App() {
   const [imagingModality, setImagingModality] = useState<FieldInput>(null);
   const [searchParams, setSearchParams] = useSearchParams();
 
+  const node: FieldInputOption[] = nodeOptions
+    .filter((n) => searchParams.getAll('node').includes(n.NodeName))
+    .map((n) => ({ label: n.NodeName, id: n.ApiURL }));
```

Comment:
maybe use a different name here, so we don't have `n` twice - that's confusing

Reply:
how about `availableNode` and `selectedNode`?

Reply:
> maybe use a different name here, so we don't have n twice - that's confusing

Shouldn't be the same since we're chaining of off the available nodes? We filter the options and then map the same filtered options.

Reply:
the same type/shape, yes. But it's a different variable name. The only thing that separates them is the scope - but it makes it tricky to read / possible to get confused